client/state/themes/selectors/is-full-site-editing-theme.ts

```diff
@@ -6,12 +6,16 @@ import 'calypso/state/themes/init';
 
 export function isFullSiteEditingTheme(
 	state: AppState,
-	themeId: string | null | undefined
+	themeId: string | null | undefined,
+	siteId: number | null | undefined
 ): boolean {
 	if ( ! themeId ) {
 		return false;
 	}
-	const theme = getTheme( state, 'wpcom', themeId ) || getTheme( state, 'wporg', themeId );
+	const theme =
+		getTheme( state, 'wpcom', themeId ) ||
+		getTheme( state, 'wporg', themeId ) ||
+		( siteId && getTheme( state, siteId, themeId ) );
```

Comment:
I wanted `isFullSiteEditingTheme` to accept `siteId` since it does not query wporg themes on "My Themes" tab (c.f. https://github.com/Automattic/wp-calypso/pull/79307).